geom/opticks_raindrop.gdml

```diff
@@ -109,16 +122,16 @@
       <fraction n="0.524" ref="Pb2080x560299383420"/>
     </element>
     <material name="G4_Pb0x5602993831c0" state="solid">
-          <property name="RINDEX" ref="RINDEX0x560299349070"/>
-
+          <property name="RINDEX" ref="RINDEX0x5602993a1df2"/>
+<property name="GROUPVEL" ref="GROUPVEL0x56029938bb90"/>
```

Comment:
Same question. Do we need to define a GROUPVEL property to see hits in this volume?

Reply:
I think they are not a must.